react/react1/week3/to-do/src/components/FetchData.jsx

```diff
@@ -0,0 +1,104 @@
+import { useEffect, useState } from "react";
+import "react-datepicker/dist/react-datepicker.css";
+import Header from "./Header";
+import TodoItem from "./ToDoItem";
+import DatePicker from "react-datepicker";
+const FetchData = () => {
```

Comment:
It is a bit unclear structurally, as this is a component called `FetchData`, but it actually does and renders a lot of other things, it is not just fetching data.

Reply:
Yes I used a improper naming convention here,thanks for correcting me .